localtypings/projectheader.d.ts

```diff
@@ -41,10 +42,13 @@ declare namespace pxt.workspace {
         isDeleted: boolean; // mark whether or not a header has been deleted
         saveId?: any; // used to determine whether a project has been edited while we're saving to cloud
 
-        // For cloud providers
-        blobId: string;       // id of the cloud blob holding this script
-        blobVersion: string;  // version of the cloud blob
-        blobCurrent: boolean; // has the current version of the script been pushed to cloud
+        // For cloud providers -- DEPRECATED
+        blobId_: string;       // id of the cloud blob holding this script
```

Comment:
note that "project id" is assigned by client and its user ID + project ID. Note "id" field